index.js

```diff
@@ -0,0 +1,122 @@
+const API_BASE = "http://content.guardianapis.com/search";
+const API_KEY = "teleporter-view";
```

Comment:
I created an API key for modern JS: 0ccb3e45-8ac1-429b-90aa-456ba71ec319

Reply:
Updated this thanks! Have you had time to have a broader look over it? Any major issues?